src/handlers/wildcard/unassign.ts

```diff
@@ -49,7 +51,21 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
   const curTimestamp = new Date().getTime();
   const lastActivity = await lastActivityTime(issue, comments);
   const passedDuration = curTimestamp - lastActivity.getTime();
-
+  const pullRequest = await getOpenedPullRequestsForAnIssue(issue.number, issue.assignee);
+
+  const response = await context.octokit.pulls.listRequestedReviewers({
```

Comment:
put it back in the helper function, just wrap it in try/catch because it will throw error so you don't need to check status. you can see other examples in helpers

Reply:
making a helper func for this?